tox.ini

```
@@ -87,12 +90,13 @@ commands =
 
 [testenv:check]
 deps =
-    docutils
     check-manifest
+    colorama  # Remove when isort > v5.12.0 is released.
```

5.11.1 should be fine, 6.0.0b2 is the problem.

```suggestion
    colorama  # TODO Remove when isort > v6.0.0b2 is released.
```